_posts/0000-01-02-amandapadgett.md.

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
I am confused; I don't see a "commit" option on this page like the instructions specify.